core/cmd/shell_local_test.go

```diff
@@ -164,6 +165,7 @@ func TestShell_RunNodeWithAPICredentialsFile(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
+			ctx := context.Background()
```

Comment:
Should use test derived contexts for these.

Reply:
I realized a little into everything we had a utility method to grab test derived contexts, and thought I had cleaned past sins up. I'll clean things up 👍